src/application-insights/azext_applicationinsights/_client_factory.py

```diff
@@ -45,3 +45,7 @@ def cf_component_billing(cli_ctx, _):
 
 def cf_api_key(cli_ctx, _):
     return applicationinsights_mgmt_plane_client(cli_ctx).api_keys
+
+
+def cf_component_linked_storage_accounts(cli_ctx, _):
```

Comment:
I didn't see you pass api version "2020-03-01-preview", how can the client be created correctly?

Reply:
It would use the default api version